src/goes/goes.py

```diff
@@ -240,41 +132,29 @@ def _string_to_abisectortype(string):
         if 'M2' in string:
```

Comment:
Are we really planning to handle incoming mesoscale region data?  The lat/lon values would need to be computed nearly daily for real-time data as the NWS picks regions of weather to suit the needs of operational forecasting.  Since I know that we are creating static lat/lon files intended to be read from disk to save time, then I hope the naming of the lat/lon static files has some identifier for full disk and conus at minimum since these should remain very nearly static always.

Reply:
This will be addressed in a future PR. Currently, we are only using full disk scans for the Data Lake project. Please remind me of this later on.